spec-0007/index.md

```diff
@@ -5,69 +5,69 @@ author:
   - "Stéfan van der Walt <stefanv@berkeley.edu>"
   - "Sebastian Berg <sebastianb@nvidia.com>"
   - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
   - Other participants in the discussion <not.yet@named.org>"
 discussion: https://github.com/scipy/scipy/issues/14322
 endorsed-by:
 ---
 
 ## Description
 
-<!--
-Briefly and clearly describe the proposal.
-Explain the general need and the advantages of this specific proposal.
-If relevant, include examples of how the new functionality would be used,
-intended use-cases, and pseudo-code illustrating its use.
--->
-
-There is disparity in the APIs libraries provide to seed random number generation.
-This SPEC suggests a single, pragmatic API for the ecosystem, taking into account technical and historical factors.
+Currently, libraries across the ecosystem provide various APIs for seeding random number generation.
```

Comment:
I think of "seeding" as the process of setting a seed. If that's right, then libraries typically get involved with "seeding" only when the `seed`/`random_state` parameter accepts a seed (e.g. integer) as an argument. 

When the `seed`/`random_state` parameter accepts a `RandomState`/`Generator`, or when the user sets a seed with `np.random.seed`, I don't think the library's API for "seeding" is being exercised.

With that in mind, consider:
```suggestion
Currently, libraries across the ecosystem provide various APIs for controlling random number generation.
```
and using "controlling" in place of "seeding" (and similar) throughout.

Reply:
I suspect "seeding", while not perfectly accurate, may convey a more intuitive meaning here.

Reply:
Hmm I thought the opposite, which was the point. But I'll leave this unmerged.

Reply:
K. I'll leave it alone and resolve.